tox.ini

```diff
@@ -3,13 +3,12 @@ skipsdist = True
 usedevelop = True
 
 envlist =
-    py{37}-dj32-wt{41,42}
     py{38,39,310}-dj{32,40,41}-wt{41,42}
-    py{311}-dj{41}-wt{41,42}
+    py{311}-dj{41}-wt{41,42,50}
+    py{311}-dj{42}-wt{50}
```

Comment:
Thanks for this. I thought Wagtail 4.1 and 4.2 are not Django 4.2 ready because of:
![image](https://github.com/jazzband/wagtailmenus/assets/26372762/96904edf-551a-4872-b27a-200235effd72)

Reply:
Also,
Can it be:
```
py{38,39,310}-dj{32,40}-wt{41,42}
py{38,39,310,311}-dj{41,42}-wt{41,42,50}
```
?

Reply:
Since we should also test Python versions 3.8-3.10 for Wagtail 5.0

Reply:
Nevermind, Django 4.2 is indeed not compatible with Wagtail 4.1 and 4.2.
Changed to:
```
py{38,39,310}-dj{32,40,41}-wt{41,42}
py{311}-dj{32,41}-wt{41,42,50}
py{38,39,310,311}-dj{42}-wt{50}
```

Please let me know if I forgot a combination or if there's a more efficient way. Thanks!